app/components/language-picker/component.js

```diff
@@ -1,40 +1,57 @@
 import Ember from 'ember';
-
+import config from 'ember-get-config';
 import countries from './countries';
 
-
 export default Ember.Component.extend({
     i18n: Ember.inject.service(),
     countries: countries,
-    languages: Ember.computed('countries', function() {
-      var langs = [];
-      this.get('countries').forEach((country) => {
-        country.languages.forEach((lang) => {
-          langs.push({
-            countryCode: country.code,
-            countryName: country.name,
-            name: lang.name,
-            code: lang.code
-          });
-        });
-      });
-      langs.sort(function(a, b) {
-        if (a.countryName > b.countryName) {
-          return 1;
-        }
-        if (a.countryName < b.countryName) {
-          return -1;
+    languages: Ember.computed('countries', 'i18n.locales', function () {
+        const {excludeTestLocale} = config.featureFlags;
+        const locales = this.get('i18n.locales').toArray();
+        const countries = this.get('countries');
+        const languages = [];
+
+        for (const locale of locales) {
+            if (excludeTestLocale && locale === 'test') {
```

Comment:
For future reference, we could probably shorten this double loop using, eg, a `Set` and `.filter`. Not sure what that would do for polyfills though.